NetworkConnectivity/src/V1/Gapic/PolicyBasedRoutingServiceGapicClient.php

```diff
@@ -235,19 +240,14 @@ public static function networkName($project, $resourceId)
      * policy_based_route resource.
      *
      * @param string $project
-     * @param string $location
      * @param string $policyBasedRoute
      *
      * @return string The formatted policy_based_route resource.
      */
-    public static function policyBasedRouteName(
-        $project,
-        $location,
-        $policyBasedRoute
-    ) {
+    public static function policyBasedRouteName($project, $policyBasedRoute)
```

Comment:
This is strange because looking at the RevId (564481387), there are no Breaking Changes being made - the entire proto for these messages ([`policy_based_routing.proto`](https://github.com/googleapis/googleapis/commits/master/google/cloud/networkconnectivity/v1/policy_based_routing.proto)) was added. They were removed in https://github.com/googleapis/googleapis/commit/183d61a81691cb57a68d1d0f871414c1cc12d526, but since removing classes is not supported in owlbot, these changes had no effect. 

The way we can fix this is to add the `policyBasedRouteName` in the wrapping `PolicyBasedRoutingServiceClient`.  The other option is to just take the breaking change, since (presumably) this API was not being used.

Reply:
Added a fix for the RPC in https://github.com/googleapis/gapic-generator-php/pull/653, but the template name still needs to be fixed (or justified) as well.

Reply:
After talking to @noahdietz, this service wasn't active until this week. So we can go ahead and take these breaking changes because it should only affect a small (or even nonexistant) number of users, and it will be a better user-experience going forward to have the libraries align with the protos.